en/components/dialog.md

```diff
@@ -4,87 +4,180 @@ _description: Whether building informative dialogs or data manipulation windows,
+
 <igx-dialog #alert
     title="Alert"
     message="Your email has been sent successfully!"
     leftButtonLabel="OK" (onLeftButtonSelect)="alert.close()">
 </igx-dialog>
 ```
-You can set title to the alert `title="TitleofTheAlert"`
 
-You can be more descriptive and set message `message="Your email has been sent successfully!"`
-
-You can attach to the left button select event `(onLeftButtonSelect)="alert.close()"`
+<div class="sample-container loading" style="height:250px">
+    <iframe id="dialog-sample-1-iframe" src='{environment:demosBaseUrl}/dialog-sample-1' width="100%" height="100%" seamless frameBorder="0" onload="onSampleIframeContentLoaded(this);"></iframe>
+</div>
+<div>
+    <button data-localize="stackblitz" class="stackblitz-btn" data-iframe-id="dialog-sample-1-iframe" data-demos-base-url="{environment:demosBaseUrl}">view on stackblitz</button>
+</div>
 <div class="divider--half"></div>
 
-###Standard Dialogs
-**Dialogs are done by adding another button.**
+####Standard Dialog
+
+To add standard dialog, in the template of our file manager component we can add the following code to get the standard dialog. We have to set the `title`, `message`, 
+`leftButtonLabel`, `rightButtonLabel`, `rightButtonRipple` and handle `onLeftButtonSelect` and `onRightButtonSelect` events:
 
 ```html
+<!--file-manager.component.html-->
+
 <igx-dialog #dialog
     title="Confirmation"
     message="Are you sure you want to delete the Microsoft_Annual_Report_2015.pdf and Microsoft_Annual_Report_2015.pdf files?"
     leftButtonLabel="Cancel"
     (onLeftButtonSelect)="dialog.close()"
     rightButtonLabel="OK"
+    rightButtonRipple="#4CAF50"
     (onRightButtonSelect)="onDialogOKSelected($event)">
 </igx-dialog>
 ```
 
-You can access all properties of the button component with the following attributes:
-
-`leftButtonLabel`
-
-`leftButtonType`
-
-`leftButtonColor`
+<div class="sample-container loading" style="height:250px">
+    <iframe id="dialog-sample-2-iframe" src='{environment:demosBaseUrl}/dialog-sample-2' width="100%" height="100%" seamless frameBorder="0" onload="onSampleIframeContentLoaded(this);"></iframe>
+</div>
+<div>
+    <button data-localize="stackblitz" class="stackblitz-btn" data-iframe-id="dialog-sample-2-iframe" data-demos-base-url="{environment:demosBaseUrl}">view on stackblitz</button>
+</div>
+<div class="divider--half"></div>
 
-`leftButtonBackgroundColor`
+####Custom Dialog
 
-`leftButtonRipple`
-<div class="divider--half"></div>
+To add custom dialog, in the template of our sign in component we can add the following code to get the custom dialog. We have to set the `title`,`leftButtonLabel`,`rightButtonBackgroundColor`, `rightButtonColor`, `rightButtonLabel`, `closeOnOutsideSelect` and handle `onLeftButtonSelect` and `onRightButtonSelect` event.
+Also we can add two groups of label and input decorated with the [**igxLabel**](../components/label_input.html) and [**igxInput**](../components/label_input.html) directives.
```

Comment:
I've now seen links done in 3 ways in the topics:
1. Production link.
2. Relative link that is just the name of the topic.
3. Relative link as is used here.
Which is correct?  Do all of them work?  Should it be consistent?

Reply:
The correct way is to use .md files. I will update these ones.